frontend/src/hooks/usePickerValues.tsx

```diff
@@ -20,11 +20,11 @@ export const formatPickerOptions = (tableData: IPickerTableData) => {
       const pickersByGroup: any = { pickers: {} };
       // todo: Define a good type. "Any" type temporarily permitted.
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      tableData.data.forEach(async (item: any) => {
-        if (!pickersByGroup.pickers[item.associated_table]) {
-          pickersByGroup.pickers[item.associated_table] = {};
+      tableData.data.data.forEach(async (item: any) => {
+        if (!pickersByGroup.pickers[item.associated_form]) {
+          pickersByGroup.pickers[item.associated_form] = {};
         }
-        pickersByGroup.pickers[item.associated_table][item.name] = item;
+        pickersByGroup.pickers[item.associated_form][item.name] = item;
```

Comment:
can we be 100% sure that this anyone of these associations will always be present.

Reply:
yes as they are apart of the seeds